src/VSCodeExtension/src/formatter/format-document.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+import { FormatRule, formatter } from "./formatter";
+import { namespaceRule } from "./rules/indent";
+import { argsRule } from "./rules/operation";
+import { spaceAfterIf } from "./rules/space-after-if";
```

Comment:
Instead of extracting each individual rule, I think it's better to expose just an array of rules to make it cleaner when there could be potentially many rules.

Reply:
If you do that, then you need to have all rules or none of the file? you would need to manually go and "unimport" the ones you don't want?

Reply:
I think we can do both, export rules indivdually or all of them to an array :) I'll do that

Reply:
I left the option of importing separated rules as well as packing them into an array for ease of use :)

Reply:
Ah I see, yea, that works. Once we get to implementing optional rules, we might change this further. Maybe passing a list of strings of rules that we don't want and then returning the resulting array of rules.